main/serve.js

```diff
@@ -16,25 +16,14 @@
  */
 'use strict';
 
-let express = require('express');
-let yargs = require('yargs');
-let path = require('path');
+require('foam2');
 
-let argv = yargs
-  .default('port', 9000)
-  .describe('port', 'Port to listen for HTTP requests on')
-  .coerce('port', (str) => parseInt(str))
-  .help()
-  .argv;
-
-let app = express();
+let server = foam.net.node.Server.create({
+  port: 9000,
+});
 
-app.use(express.static('static/bundle'));
+server.exportFile('/', `${__dirname}/../static/index.html`);
 
-app.get('/', function(req, res) {
-  res.sendFile(path.join(__dirname, '../', 'static', 'index.html'));
-});
+server.exportDirectory('/', `${__dirname}/../static/bundle`);
```

Comment:
Yes, it works.
> Handlers handle requests in the order they appear in the array.

When '/' is handled by the fileHandler, it is not passed to the staticFileHandler. There is no conflict.